logger.go

```diff
@@ -8,8 +8,13 @@ import (
 func initLogger(logPath *string, id string, debug *bool) (*zap.Logger, error) {
 
 	cfg := zap.NewProductionConfig()
-	cfg.ErrorOutputPaths = []string{*logPath + "/logger.err"}
-	cfg.OutputPaths = []string{*logPath + "/glutton.log"}
+	if *logPath != "/dev/null" {
```

Comment:
I'm still not super happy with this solution... Maybe call it log file and error log file? Hard coding the file names seems a bit restrictive and maybe counter-intuitive.